buildSrc/src/main/kotlin/Libs.kt

```diff
@@ -70,4 +71,12 @@ object Libs {
 
     // GLIDE
     const val GLIDE = "com.github.bumptech.glide:glide:${Versions.GLIDE}"
+
+    // Firebase-BOM
+    const val BOM = "com.google.firebase:firebase-bom:${Versions.BOM}"
+    const val FIREBASE_SDK_ANALYTICS = "com.google.firebase:firebase-analytics-ktx"
+    const val FIREBASE_AUTH = "com.google.firebase:firebase-auth-ktx:${Versions.FIREBASE_KTX}"
```

Comment:
If you're using BOM then you don't need to specify version. Seems you didn't followed documentation properly.

https://firebase.google.com/docs/android/setup#add-sdks

Reply:
Yes , removed the firebase-auth version.